en/examples/fly_mission.md

```diff
@@ -81,36 +81,14 @@ if(NOT MSVC)
     add_definitions("-std=c++11 -Wall -Wextra -Werror")
```

Comment:
All these example files still don't run. When they do, will run and update the print out log to current version.